### ISSUE TITLE
Load WAV files

### ISSUE THREAD

**Issue by ataulien** [REGoth-project/REGoth-bs#93]
The games sounds come as regular `.WAV` files which are stored within the VDFS. Actual sound *effects* should even be uncompressed, while dialogue lines use Microsoft ADPCM. 

If bsf cannot load MSADPCM, a new issue should be made for that. Loading uncompressed WAV should be done first.
